capa/features/extractors/viv/extractor.py

```diff
@@ -20,21 +20,22 @@
 from capa.features.common import Feature
 from capa.features.address import Address, AbsoluteVirtualAddress
 from capa.features.extractors.base_extractor import BBHandle, InsnHandle, FunctionHandle, FeatureExtractor
-
+from capa.features.extractors.strings import DEFAULT_STRING_LENGTH
```

Comment:
where is this used?

Reply:
Changed constructor to take the value as a default.